api/WebSocket.json

```diff
@@ -1050,7 +1050,7 @@
               "version_added": "5"
             },
             "safari_ios": {
-              "version_added": "5.1"
+              "version_added": "5"
```

Comment:
Are you sure this shouldn't be 4.2? (I can't find a previous thread, maybe I missed this in the first round.)

Reply:
This one's for worker support, which was introduced in Safari iOS 5.  The rest of the API is set to 4.2, which I've just double-checked is accurate!

Reply:
Ah, thanks!